actions/installer.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
nit: While not totally necessary we've tried to get away from writing a lot of bash for actions. Is there any way this could be written in TypeScript?

See the compute-sha256 actions in the slsa-github-generator repo for an example.
https://github.com/slsa-framework/slsa-github-generator/tree/main/.github/actions/compute-sha256